coverage_executor.go

```diff
@@ -2,30 +2,34 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-tools/go-steputils/tools"
-	"os"
-	"path/filepath"
 )
 
 const (
-	coverageFileName       = "flutter_coverage_lcov.info"
-	coveragePath           = "./coverage/lcov.info"
+	coverageFileName     = "flutter_coverage_lcov.info"
+	coverageRelativePath = "./coverage/lcov.info"
 )
 
 type coverageExecutor interface {
-	executeCoverage(additionalParams []string) bool
-	exportCoverage()
+	executeCoverage(cfg config, additionalParams []string) bool
+	exportCoverage(projectLocation string)
 }
 
 type realCoverageExecutor struct {
 	interrupt      interrupt
 	commandBuilder commandBuilder
 }
 
-func (r realCoverageExecutor) executeCoverage(additionalParams []string) bool {
-	coverageCmdModel := r.commandBuilder.buildCoverageCmd(additionalParams)
+func (r realCoverageExecutor) executeCoverage(cfg config, additionalParams []string) bool {
```

Comment:
why do we pass the whole config? why not just the ProjectLocation?

Reply:
Yeah you are right, we should just pass the location.